mule-migration-tool-engine/src/main/java/com/mulesoft/tools/migration/engine/project/structure/ApplicationPersister.java

```diff
@@ -89,6 +93,32 @@ public void persist() throws Exception {
       persistMuleArtifactJson();
       persistPom();
       persistMuleAppProperties();
+      persistGitIgnoreFile();
+    }
+  }
+
+  /**
+   * persist a .gitIgnore File from gitignore-maven-template
+   * 
+   * @throws FileNotFoundException in case of a FileNotFoundException
+   * @throws IOException in case of a IOException
+   */
+  private void persistGitIgnoreFile() throws FileNotFoundException, IOException {
+
+    if (projectOutput instanceof MuleProject) {
+      MuleProject muleProject = (MuleProject) projectOutput;
+      Path baseFolder = muleProject.baseFolder;
+      if (baseFolder.toFile().exists()) {
+        InputStream gitignoreResourceStream = null;
+        try {
+          gitignoreResourceStream = getClass().getClassLoader().getResourceAsStream("gitignore-maven-template");
+          File gitIgnore = new File(baseFolder.toFile(), ".gitignore");
+          OutputStream outputStream = new FileOutputStream(gitIgnore);
+          IOUtils.copy(gitignoreResourceStream, outputStream);
+        } finally {
+          IOUtils.closeQuietly(gitignoreResourceStream);
```

Comment:
I think we should close the outputstream as well here, not only the inputstream.

Reply:
outputstream also closed now